### DISCUSSION TITLE
Lucene cross class index on 3.1/3.2

### DISCUSSION THREAD

**Discussion by timw** [orientechnologies/orientdb#9849] [Ideas] [answered]
It appears that the enterprise agent is being built and published for 3.1.x and 3.2.x [1], but the `lucene-cross-class` module [2] of the agent is only built for 3.0.x [3].

Would it be possible to have this built for 3.0.x/3.1.x, or alternatively would a PR to incorporate the cross_class index/function into the OrientDB mainline be accepted?

[1]: https://repo1.maven.org/maven2/com/orientechnologies/agent/
[2]: https://github.com/SAP/orientdb-enterprise-agent/tree/develop/lucene-cross-class
[3]: https://repo1.maven.org/maven2/com/orientechnologies/orientdb-lucene-cross-class/

**Comment by tglman** [comment, marked as answer]
Hi,

I think include the `lucene-cross-class` would be possible, here is the repository that contains the code of the agent https://github.com/SAP/orientdb-enterprise-agent   in there is also the `lucene-cross-class` I guess it's all down to the build configuration, contributions are welcome for the reintroduction.

**Reply by timw** [reply]
I've created #9855 to port `SEARCH_CLASS` from the enterprise edition.
